linodecli/plugins/k8s-alpha.py

```diff
@@ -0,0 +1,286 @@
+    },
+    'master_type': {
+        'name': 'server_type_master',
+        'default': 'g6-standard-2',
+    },
+    'region': {
+        'name': 'region',
+        'default': 'us-west',
+    },
+    'ssh_private_key': {
+        'name': 'ssh_private_key',
+        'default': os.path.expanduser('~/.ssh/id_rsa'),
+    },
+    'ssh_public_key': {
+        'name': 'ssh_public_key',
+        'default': os.path.expanduser('~/.ssh/id_rsa.pub'),
+    },
+}
+
+def create(args, context):
+    parser = argparse.ArgumentParser("{} create".format(plugin_name), add_help=True)
+    parser.add_argument('name', metavar='NAME', type=str,
+                        help="A name for the cluster.")
+#    High availability master nodes coming soon.
+#    parser.add_argument('--ha', metavar="MASTERS", type=int, required=False,
+#                        choices=[3, 5],
+#                        help="Make the cluster highly-available with MASTERS "
+#                             "number of masters (3 or 5)")
+    parser.add_argument('--node-type', metavar="TYPE", type=str, required=False,
+                        default=tf_var_map['node_type']['default'],
+                        help='Linode Type ID for cluster Nodes as retrieved with '
+                             '`linode-cli linodes types`. (default "g6-standard-2")')
+    parser.add_argument('--nodes', metavar="COUNT", type=int, required=False,
+                        default=tf_var_map['nodes']['default'],
+                        help='The number of Linodes to deploy as Nodes in the cluster. '
+                             '(default 3)')
+    parser.add_argument('--master-type', metavar="TYPE", type=str, required=False,
+                        default=tf_var_map['master_type']['default'],
+                        help='Linode Type ID for cluster Master Nodes as retrieved with '
+                             '`linode-cli linodes types`. (default "g6-standard-2")')
+    parser.add_argument('--region', metavar="REGION", type=str, required=False,
+                        default=tf_var_map['region']['default'],
+                        help='The Linode Region ID in which to deploy the cluster as retrieved with '
+                             '`linode-cli regions list`. (default "us-west")')
+    parser.add_argument('--ssh-private-key', metavar="KEYPATH", type=str, required=False,
+                        default=tf_var_map['ssh_private_key']['default'],
+                        help='The path to your private key file which will be used to access Nodes')
+    parser.add_argument('--ssh-public-key', metavar="KEYPATH", type=str, required=False,
+                        default=tf_var_map['ssh_public_key']['default'],
+                        help='The path to your public key file which will be used to access Nodes')
+    parsed, _ = parser.parse_known_args(args)
+
+    # Check if Terraform is installed
+    try:
+        nullf = open(os.devnull, 'w')
+        spcall(['terraform'], stdout=nullf)
```

Comment:
you can import `DEVNULL` from `subprocess` for `stdout` instead of doing this

Reply:
In subprocess `DEVNULL = -3`, in other operating environments, the "null file descriptor" may not be -3. This is more platform independent.

Reply:
other operating environments are stupid

Reply:
can we context manage this one too, in that case?